ENIGMAsystem/SHELL/Platforms/Win32/Makefile

```diff
@@ -1,2 +1,3 @@
 SOURCES += $(wildcard Platforms/Win32/*.cpp)
+override CXXFLAGS += -DENIGMA_PTFRM_WIN
```

Comment:
You missed an L. At this point, why not just `ENIGMA_PLATFORM_WINDOWS`? It's the trend we've been moving toward elsewhere (just say what you mean so I don't have to remember your weird abbreviations).